swift/test/Ice/udp/AllTests.swift

```diff
@@ -91,11 +71,12 @@ public func allTests(_ helper: TestHelper) async throws {
         var seq: ByteSeq
         do {
             seq = ByteSeq(repeating: 0, count: 1024)
+            let replyI = PingReplyI()
+            let reply = try replyI.getProxy(adapter)
+
             while true {
                 seq = ByteSeq(repeating: 0, count: seq.count * 2 + 10)
-                replyI.reset()
                 try await obj.sendByteSeq(seq: seq, reply: reply)
-                _ = replyI.waitReply(expectedReplies: 1, timeout: 10000)
```

Comment:
We're looping until an invocation throws DatagramLimitException. What's the point of this "waitReply" call that doesn't throw?